html/pfappserver/lib/pfappserver/Base/Form.pm

```diff
@@ -171,6 +176,47 @@ sub id_validator {
    };   
 }
 
+before 'process' => sub {
+    my ($self) = @_;
+    foreach my $field (@{$self->fields}){
+        # Populate all the FingerbankSelect fields with the proper values extracted from the database.
+        if($field->type eq "FingerbankSelect"){
```

Comment:
Because the options differ form the pre and post processing since options are added dynamically to the options fields based on what is selected in order to prevent putting the 6000 Fingerbank devices as an option